DQMOffline/Configuration/python/autoDQM.py

```diff
@@ -1,18 +1,22 @@
-autoDQM = { 'common': ['@dcs+DQMMessageLoggerSeq+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@castor+@physics+@tau',
+autoDQM = { 'DQMMessageLogger': ['DQMMessageLoggerSeq',
+                              'PostDQMOffline',
+                              'DQMMessageLoggerClientSeq'],
+
+            'common': ['@dcs+@DQMMessageLogger+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@castor+@physics+@tau',
                         'PostDQMOffline',
-                        '@dcs+DQMMessageLoggerClientSeq+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@fed+@tau+dqmFastTimerServiceClient'],
+                        '@dcs+@DQMMessageLogger+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@fed+@tau+dqmFastTimerServiceClient'],
 
-            'commonFakeHLT': ['@dcs+DQMMessageLoggerSeq+@strip+@pixel+@tracking+@L1TMon+@beam+@castor+@physics+@tau',
+            'commonFakeHLT': ['@dcs+@DQMMessageLogger+@strip+@pixel+@tracking+@L1TMon+@beam+@castor+@physics+@tau',
                         'PostDQMOffline',
-                        '@dcs+DQMMessageLoggerClientSeq+@strip+@pixel+@tracking+@L1TMon+@beam+@fed+@tau+dqmFastTimerServiceClient'],
+                        '@dcs+@DQMMessageLoggerClient+@strip+@pixel+@tracking+@L1TMon+@beam+@fed+@tau+dqmFastTimerServiceClient'],
 
-            'commonSiStripZeroBias': ['@dcs+DQMMessageLoggerSeq+DQMOfflineCommonSiStripZeroBias+@tracking+@L1TMon+@hlt+@beam+@castor+@physics',
+            'commonSiStripZeroBias': ['@dcs+@DQMMessageLogger+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@castor+@physics',
```

Comment:
@jfernan2 why you substitute here :
`DQMOfflineCommonSiStripZeroBias` with `@strip+@pixel`?
Shouldn't it be `@stripZeroBias+@pixel` instead?

Reply:
I guess Marco is right here, please @jfernan2 have a look

Reply:
@jfernan2 I had noticed the same when looking at the crash, but in itself it was not sufficient to prevent it

Reply:
Sorry, my mistake, I was working with two solutions in parallel and mixed them up. Apologies for the inconvenience